VM/plot_vm.py

```diff
@@ -0,0 +1,108 @@
+import os
```

Comment:
This script can easily have a main function that generates the plot from the vm_params.yaml file, realisation csv file (We can recalculate the corners from that data) and argparse

Reply:
Not straightforward as plotting is closely tied to the extended vm_params taken from the optimisation. Made a bit of hack that works out the missing values and feed to the plot_vm. Skips SRF-related info, but serves the purpose.